tutor/config.py

```diff
@@ -4,6 +4,8 @@
 from . import env, exceptions, fmt, plugins, serialize, utils
 from .types import Config, cast_config
 
+_CONFIG_FILE = "config.yml"
```

Comment:
If this variable is meant to be imported elsewhere, let's not prefix it with an underscore. Also, it's a filename, so let's name it `CONFIG_FILENAME`.